.github/workflows/java-build.yml

```
@@ -0,0 +1,28 @@
+name: Java (build)
+
+on:
+  push:
+    paths:
+      - 'visual-java/**'
```

```suggestion
      - 'visual-java/**'
      - .github/workflows/java-build.yml
```
So the build will be kicked if we only modify the workflow (and won't discover that it fails upon merge)